qtconsole/console_widget.py

```diff
@@ -1872,6 +1874,44 @@ def _get_word_end_cursor(self, position):
             cursor.setPosition(position)
         return cursor
 
+    def _indent(self, dedent=True):
+        """ Indent/Dedent current line or current text selection.
+        """
+        num_newlines = self._get_num_lines_selection()
+        save_cur = self._get_cursor()
+        cur = self._get_cursor()
+
+        # - move to first line of selection, if present
+        # - calculate number of space to indent it to 4-space multiple
+        cur.setPosition(cur.selectionStart())
+        self._control.setTextCursor(cur)
+        spaces = self._get_leading_spaces()
```

Comment:
# Calculate number of space to indent it to 4-space multiple
    spaces = self._get_leading_spaces()

Reply:
Fixed